### PR TITLE
Add cbrt to jax frontend api with test

### LINKED ISSUE #20875: cbrt

### PR DESCRIPTION
**Pull request by Sjsingh101**
closes #20875

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py [hunk 1]
```diff
@@ -2931,3 +2931,32 @@ def test_jax_is_finite(
         on_device=on_device,
         x=x[0],
     )
+
+
+# cbrt
+@handle_frontend_test(
+    fn_tree="jax.lax.cbrt",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
```
Comment (hello-fri-end) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py:2940]:
```suggestion
        available_dtypes=helpers.get_dtypes("valid")
```
Valid automatically determines the supported dtypes for the function.

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @Sjsingh101 ! Thanks for creating this PR. Just left a minor comment. Feel free to request another review when you have resolved it :smile:

**Comment by Sjsingh101** [comment]
Hi @hello-fri-end, I have updated the PR but many other test (like `test_jax_square`, `test_jax_add`, `test_jax_sub`, etc) operate on similar input types and have "numeric" dtypes. Should I update them as well?

File: ivy/functional/frontends/jax/lax/operators.py [hunk 2]
```diff
@@ -647,3 +647,8 @@ def conj(x):
 @to_ivy_arrays_and_back
 def is_finite(x):
     return ivy.isfinite(x)
+
+
+@to_ivy_arrays_and_back
+def cbrt(x):
```
Comment (hello-fri-end) [inline ivy/functional/frontends/jax/lax/operators.py:653]: Can you add the `/` argument here as it's present in the `jax.cbrt` function here :https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.cbrt.html

**Review by hello-fri-end — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/lax/operators.py (hunk 2 above)
Comment (hello-fri-end) [reply ivy/functional/frontends/jax/lax/operators.py:658]: Oh awesome, thanks for the explanation  :fire:

**Review by hello-fri-end — APPROVED** [review]
Looks like the tests are passing now. Merging this :rocket: Thanks for the contribution :partying_face:

**Pull request merged**